programs/ssl/ssl_server2.c

```diff
@@ -1791,7 +1791,7 @@ int main( int argc, char *argv[] )
             unsigned char *larger_buf;
 
             ori_len = ret;
-            extra_len = ssl_get_bytes_avail( &ssl );
+            extra_len = (int) ssl_get_bytes_avail( &ssl );
```

Comment:
I think both:)
I think you should fix here, as we are already modifying the file, and raise another issuer for the other two branches